hr_check_ssnid/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{'name': 'France - Ssnid check',
+ 'version': '10.0.1.0.0',
+ 'author': 'Camptocamp,Odoo Community Association (OCA)',
+ 'license': 'AGPL-3',
+ 'category': 'French Localization',
+ 'depends': ['hr',
+             ],
+ 'website': 'https://www.camptocamp.com',
+ 'data': ['views/hr_employee.xml',
+          ],
+ 'installable': True,
+ 'external_dependencies': {'python': ['stdnum']}
```

Comment:
also add `stdnum` to `requirements.txt` file in repository root.
If the file isn't there, create it.

Reply:
in other OCA repo I didn't see `requirement.txt` link https://github.com/OCA/geospatial/blob/12.0/base_geoengine_demo/__manifest__.py#L22

https://github.com/OCA/geospatial

Reply:
@Tonow-c2c but if it exists it'll be read by the Travis configuration (e.g. https://github.com/OCA/report-print-send/)

Reply:
Thanks :+1: